retail/interactive-tutorial/RetailEvents.Samples/RejoinUserEventSample.cs

```diff
@@ -0,0 +1,128 @@
+﻿// Copyright 2021 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START retail_rejoin_user_event]
+// Import user events into a catalog from inline source using Retail API
+
+using Google.Cloud.Retail.V2;
+using Google.LongRunning;
+using Google.Protobuf.WellKnownTypes;
+using System;
+
+/// <summary>
+/// The rejoin user event sample class.
+/// </summary>
+public class RejoinUserEventSample
+{
+    /// <summary>
+    /// Get the user event.
+    /// </summary>
+    /// <returns>User event object.</returns>
+    public static UserEvent GetUserEvent()
+    {
+        ProductDetail productDetail = new ProductDetail
+        {
+            Product = new Product
+            {
+                Id = "test_id"
+            },
+            Quantity = 3
+        };
+
+        UserEvent userEvent = new UserEvent
+        {
+            EventType = "detail-page-view",
+            VisitorId = "test_visitor_id",
+            EventTime = new Timestamp
```

Comment:
same commen as before on how to build the Timestamp

Reply:
Fixed.